less/list.less

```diff
@@ -44,14 +41,11 @@
       width: 15%;
     }
   }
-  // --------------------------------------------------
 }
 
-// --------------------------------------------------
 // --------------------------------------------------
```

Comment:
Should this line need to be removed? Not sure of your rules.

Reply:
The general comment rule of thumb is

```
// Comment
// --------------------------------------------------
```

```
// --------------------------------------------------
// Divider
// --------------------------------------------------
```